### PR TITLE
Assignment 5

### PR DESCRIPTION
**Pull request by pablocabargas**

### CODE REVIEW

File: Assignment5/pablocabargas-21A328/assignment_notebook.py [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+      ?member wdt:P31 wd:Q5.
+      ?member rdfs:label ?label.
+      FILTER(LANG(?label) = 'en').
+}  """
+
+
+# Kypher:
+kgtk("""
+    query -i all
+    --match '(:Q2685)-[:P102]->(party)-[:P488]->(persons_of_chair)-[:P31]->(:Q5)'
```
Comment (dgarijo) [inline Assignment5/pablocabargas-21A328/assignment_notebook.py:181]: Why P488? It should be P102 again...

File: Assignment5/pablocabargas-21A328/assignment_notebook.py [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+      ?x ?property ?v.
+} 
+"""
+
+
+
+# In Kypher:
+kgtk("""
+    query -i all
+    --match '()<-[p {label: property}]-()-[:P279*]->(:Q483501)'
```
Comment (dgarijo) [inline Assignment5/pablocabargas-21A328/assignment_notebook.py:205]: Not correct. You are showing the properties of the class, not from the individuals with that profession (P106)

File: Assignment5/pablocabargas-21A328/assignment_notebook.py [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+WHERE {
+      ?x wdt:P106 wd:Q2526255.
+      ?x ?property ?v.     
+} 
+"""
+
+# In Kypher:
+kgtk("""
+    query -i all
+    --match '()<-[p {label: property}]-()-[:P279*]->(:Q2526255)'
```
Comment (dgarijo) [inline Assignment5/pablocabargas-21A328/assignment_notebook.py:222]: Same as above. It should be `'(o)<-[l {label: property}]-(s)-[:P106]->(:Q2526255)`

Comment (dgarijo) [reply Assignment5/pablocabargas-21A328/assignment_notebook.py:222]: And optionally `(o)<-[l {label: property}]-(s)-[:P106]->(:Q2526255)-[:P279*]->(:Q2526255)`

**Comment by dgarijo** [comment]
Assignment 5 has errors, but not big ones. However, this PR is changing other files, so I won't merge it (@ocorcho  to have a look)

**Comment by pablocabargas** [comment]
Hi Daniel, i also commented on the previous PR of assignment 4, if this could be merged or not? best regards

**Pull request merged**